infiniband-diags/CMakeLists.txt

```diff
@@ -8,16 +8,13 @@ install(FILES
   etc/ibdiag.conf
   DESTINATION "${IBDIAG_CONFIG_PATH}")
 
-add_library(ibdiags_tools STATIC
```

Comment:
The library prevents multiple recompilation, why can't we fix the library?

Reply:
It doesn't worth, I see "fixing" library as optimization of something that no one cares.

Reply:
BTW, I fixed it locall by putting ibdiag_tools as first in the library list.

Just can't push till we fix the suse failure.